### PR TITLE
Allow authors to vote on community tag

### PR DESCRIPTION
**Pull request by oetherington**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204097489951677) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by s-cheng** [comment]
@oetherington Sorry about the conflicts :( I'll assign this back to you for now

**Comment by jpaddison3** [comment]
Wait what? But then authors can strong downvote to remove the community tag, which is v bad?

**Comment by oetherington** [comment]
@s-cheng This should be ready for review again now - sorry for the wait!

File: packages/lesswrong/lib/collections/tags/schema.ts [hunk 1]
```diff
@@ -471,8 +472,8 @@ const schema: SchemaType<DbTag> = {
     group: formGroups.advancedOptions,
   },
   'canVoteOnRels.$': {
-    // TODO: Enum, allowed values: 'userOwns', or a user group name, also: 'userOwnsOnlyUpvote'
     type: String,
+    allowedValues: ["userOwns", "userOwnsOnlyUpvote", ...permissionGroups],
```
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/tags/schema.ts:476]: Woah

File: packages/lesswrong/components/posts/PostsPage/PostsPage.tsx [hunk 2]
```diff
@@ -344,6 +345,7 @@ const PostsPage = ({post, refetch, classes}: {
   }
 
   return (<AnalyticsContext pageContext="postsPage" postId={post._id}>
+    <PostsPageContext.Provider value={post}>
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsPage/PostsPage.tsx:348]: Seems reasonable.

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Pull request merged**